diff --git a/src/SiftGPU/CMakeLists.txt b/src/SiftGPU/CMakeLists.txt
index aaa0d11..29c47a6 100644
--- a/src/SiftGPU/CMakeLists.txt
+++ b/src/SiftGPU/CMakeLists.txt
@@ -41,7 +42,7 @@ if (GLEW_FOUND)
 	SET (GLEW_LIBRARIES GLEW)
 endif (GLEW_FOUND)
 
-ADD_LIBRARY(siftgpu STATIC FrameBufferObject.cpp GlobalUtil.cpp GLTexImage.cpp ProgramGLSL.cpp 
+ADD_LIBRARY(siftgpu SHARED FrameBufferObject.cpp GlobalUtil.cpp GLTexImage.cpp ProgramGLSL.cpp 
             ProgramGPU.cpp ShaderMan.cpp SiftGPU.cpp SiftPyramid.cpp PyramidGL.cpp SiftMatch.cpp)
 
 TARGET_LINK_LIBRARIES(siftgpu ${LIBS_SIFTGPU} ${GLEW_LIBRARIES} ${GLUT_LIBRARIES} ${OPENGL_LIBRARIES} )
